CHANGELOG.md

```diff
@@ -1,6 +1,29 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## [0.6.5] - 2024-11-08
+### Added
+  * Update to Golang 1.22.
+  * Implement 'relaxed-read-only' mode to ease the bind-mount requirements of read-only containers.
+  * Enhance sysbox-fs emulation to allow writes to '/proc/sys/kernel/shm*' paths.
+  * Fix emulation of /proc/sys/kernel/random (sysbox issue #785).
```

Comment:
nit: replace "sysbox issue ..." with just "issue ..." (as in prior changelog releases).